geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/DelimitedIngest.scala

```diff
@@ -83,46 +89,32 @@ class DelimitedIngest(params: IngestParameters) extends AccumuloProperties {
   def ingestJarSearchPath: Iterator[() => Seq[File]] =
     Iterator(() => ClassPathUtils.getJarsFromEnvironment("GEOMESA_HOME"),
       () => ClassPathUtils.getJarsFromEnvironment("ACCUMULO_HOME"),
-      () => ClassPathUtils.getJarsFromClasspath(classOf[ScaldingDelimitedIngestJob]),
+      () => ClassPathUtils.getJarsFromClasspath(classOf[ScaldingConverterIngestJob]),
       () => ClassPathUtils.getJarsFromClasspath(classOf[AccumuloDataStore]),
       () => ClassPathUtils.getJarsFromClasspath(classOf[Connector]))
 
   def getScaldingArgs(): Args = {
-    val singleArgs = List(classOf[ScaldingDelimitedIngestJob].getCanonicalName, getModeFlag(params.files(0)))
+    val singleArgs = List(classOf[ScaldingConverterIngestJob].getCanonicalName, getModeFlag(params.files(0)))
 
+    val sftString = SimpleFeatureTypes.encodeType(sft)
     val requiredKvArgs: Map[String, List[String]] = Map(
       IngestParams.FILE_PATH         -> encodeFileList(params.files.toList),
-      IngestParams.SFT_SPEC          -> URLEncoder.encode(params.spec, "UTF-8"),
+      IngestParams.SFT_SPEC          -> URLEncoder.encode(sftString, StandardCharsets.UTF_8.displayName),
       IngestParams.CATALOG_TABLE     -> params.catalog,
       IngestParams.ZOOKEEPERS        -> Option(params.zookeepers).getOrElse(zookeepersProp),
       IngestParams.ACCUMULO_INSTANCE -> Option(params.instance).getOrElse(instanceName),
       IngestParams.ACCUMULO_USER     -> params.user,
       IngestParams.ACCUMULO_PASSWORD -> getPassword(params.password),
-      IngestParams.DO_HASH           -> params.hash.toString,
-      IngestParams.FORMAT            -> Option(params.format).getOrElse(getFileExtension(params.files(0))),
-      IngestParams.FEATURE_NAME      -> params.featureName,
-      IngestParams.IS_TEST_INGEST    -> false.toString
+      IngestParams.ACCUMULO_MOCK     -> params.useMock.toString,
+      IngestParams.FEATURE_NAME      -> sft.getTypeName,
+      IngestParams.IS_TEST_INGEST    -> false.toString,
+      IngestParams.CONVERTER_CONFIG  -> URLEncoder.encode(converterConfig.root().render, StandardCharsets.UTF_8.displayName)
```

Comment:
you can pass in ConfigRenderOptions.concise() to render to strip out comments and whitespace and save a little space